### PR TITLE
Feweh(????)'s Brothel Changes

### PR DESCRIPTION
**Pull request by GhostEcho**
The Brothel has been renovated, and taken over by the Omerta family, redesigning it in a similar fashion to their Gomorrah casino located in New Vegas
Brothel_end

### CODE REVIEW

**Comment by improvedname** [comment]
Table oe go home

**Comment by LetterN** [comment]
testing mapdiffbot

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -51,47 +51,48 @@
...
-"abb" = (/obj/machinery/light/small{brightness = 3;dir = 8},/turf/open/floor/plasteel/f13{icon_state = "dark"},/area/f13/legioncamp)
+"abb" = (/obj/machinery/light/small{brightness = 3; dir = 8},/turf/open/floor/plasteel/f13{icon_state = "dark"},/area/f13/legioncamp)
 "abc" = (/mob/living/simple_animal/cow/brahmin,/turf/open/floor/plating/f13/outside/desert,/area/f13/desert)
 "abd" = (/obj/structure/sink/well{icon_state = "well_3"},/turf/open/floor/plating/f13/outside/desert,/area/f13/desert)
 "abe" = (/obj/structure/sink/well{icon_state = "well_4"},/turf/open/floor/plating/f13/outside/desert,/area/f13/desert)
 "abf" = (/obj/machinery/door/unpowered/wooddoor,/turf/open/floor/plating/f13/inside/mountain,/area/f13/underground/mountain)
 "abg" = (/turf/closed/wall/r_wall/f13composite{icon_state = "ruins1"},/area/f13/desert)
+"abh" = (/turf/open/floor/plating/f13/outside/desert,/obj/docking_port/stationary{dir = 8; dwidth = 3; height = 15; id = "arrivals_stationary"; name = "arrivals"; roundstart_template = /datum/map_template/shuttle/arrival/box; width = 7},/turf/open/floor/plating/f13/inside/mountain,/area/f13/underground/mountain)
```
Comment (LetterN) [inline _maps/map_files/BoxStation/BoxStation.dmm]: aaaa test latejoins

**Comment by UndeadNappist** [comment]
Part of the charm of the brothel and a lot of buildings is to leave them half finished, busted, or deliberately ugly so people fix them up. This redesign is much more appropriate for the **Den** casino, and not the ruined sunnydale brothel.

**Pull request closed without merging**